### PR TITLE
Added file picker input control

### PR DESCRIPTION
**Pull request by jamesmcroft**
- [ ] Bug fix
- [x] Feature/change request
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build or CI related changes
- [ ] Other (include details)

Added a new `FilePicker` input control similar to the web input equivalent. 

Allows the user to select single or multiple files, the ability to append to the current selection to allow subsequent adds, and the ability to view the files with a preview thumbnail and file name in a `GridView`.

- [ ] Tests have been added/updated (where applicable) and pass
- [x] Code styling has been run on all new source file changes
- [ ] Documentation has been added
- [x] Code sample has been added
- [x] Contains **NO** breaking changes

### CODE REVIEW

**Comment by jamesmcroft** [comment]
@tom-made added more changes to the new file picker element. The functionality includes the ability to remove a chosen item.

File: src/MADE.Media.Image/MADE.Media.Image.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <TargetFramework>uap10.0.17763</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Product>MADE.NET Images</Product>
+    <Description>
+      This package includes image helpers such as:
+      - StorageFileThumbnailConverter for providing the ability to show a thumbnail for a storage file.
```
Comment (tom-made) [inline src/MADE.Media.Image/MADE.Media.Image.csproj:9]: Is this meant to be `LoadStorageFileThumbnailImageBehavior`?

File: src/MADE.Media.Image/MADE.Media.Image.csproj [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    </Description>
+    <PackageTags>MADE UI Views Controls UWP FilePicker Input File</PackageTags>
```
Comment (tom-made) [inline src/MADE.Media.Image/MADE.Media.Image.csproj:11]: These package tags look like they were copied over from the `FilePicker` lib. Worth updating these.

File: src/MADE.UI.Controls.FilePicker/FilePickerItemClickEventArgs.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+namespace MADE.UI.Controls
```
Comment (tom-made) [inline src/MADE.UI.Controls.FilePicker/FilePickerItemClickEventArgs.cs:1]: Missing doc headers.

**Review by tom-made — CHANGES_REQUESTED** [review]
Looks good, just a couple of changes needed

File: src/MADE.Media.Image/MADE.Media.Image.csproj (hunk 1 above)
Comment (jamesmcroft) [reply src/MADE.Media.Image/MADE.Media.Image.csproj]: Sure was, should be sorted and updated the description.

File: src/MADE.Media.Image/MADE.Media.Image.csproj (hunk 2 above)
Comment (jamesmcroft) [reply src/MADE.Media.Image/MADE.Media.Image.csproj]: Updated the tags to reflect the library

File: src/MADE.UI.Controls.FilePicker/FilePickerItemClickEventArgs.cs (hunk 3 above)
Comment (jamesmcroft) [reply src/MADE.UI.Controls.FilePicker/FilePickerItemClickEventArgs.cs:4]: This should be sorted now

**Review by tom-made — APPROVED** [review]
Looks good 👍

**Pull request merged**